src/01.UI-Elements/ListItem.js

```diff
@@ -54,17 +55,26 @@ export default function ListItem({
     e.target.blur()
     onEnter()
   }
+
+  function showDeleteIcon() {
+    setIsDeleteIconShown(true)
+  }
+
+  function hideDeleteIcon() {
+    let raceConditionTimer = setTimeout(() => setIsDeleteIconShown(false), 0)
+    return () => clearTimeout(raceConditionTimer)
+  }
 }
 
 const ListItemStyled = styled.label`
-  width: 80vw;
+  width: 75vw;
   display: flex;
   align-items: center;
-  text-decoration: ${(props) => (props.checked ? 'line-through' : 'none')};
   opacity: ${(props) => (props.checked ? 0.5 : 1)};
 `
 const TitleStyled = styled.input`
   margin-left: 9px;
+  //text-decoration: ${(props) => (props.checked ? 'line-through' : 'none')};
```

Comment:
You're seem to be unsure if you want to give it in or to have it here directly in the style. Don't see the problem to have it here :)